src/Logging/Logging.Abstractions/src/NullLoggerFactory.cs

```diff
@@ -9,6 +9,14 @@ namespace Microsoft.Extensions.Logging.Abstractions
     /// </summary>
     public class NullLoggerFactory : ILoggerFactory
     {
+        /// <summary>
+        /// Creates a new <see cref="NullLoggerFactory"/> instance.
+        /// </summary>
+        public NullLoggerFactory() { }
```

Comment:
Weird. I feel like this was meant to be private, but wasn't. It'd be a breaking change to make it private. Should it be private? And if so, worth the break?

Reply:
If you want to file an issue I think that's a reasonable change, but since this is a docs PR I'm trying to keep functional changes out of it.

Reply:
Ah, looks like there's already a bug on that: https://github.com/aspnet/Extensions/issues/662. I agree it shouldn't be done as part of this PR; it was merely an observation.